chromium_src/chrome/common/chrome_features.cc

```diff
@@ -28,6 +28,7 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
     {kShortcutsNotApps, base::FEATURE_DISABLED_BY_DEFAULT},
 #endif
+    {kDesktopPWAsTabStripSettings, base::FEATURE_ENABLED_BY_DEFAULT},
```

Comment:
Can we add an entry to `FeatureDefaultsTest.EnabledFeatures` to verify this default?